recipe/meta.yaml

```
@@ -1,49 +1,64 @@
-{% set version = "0.8.29" %}
+{% set version = "0.8.33" %}
+{% set name = "hdbscan" %}
 
 package:
-  name: hdbscan
+  name: {{ name }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/h/hdbscan/hdbscan-{{ version }}.tar.gz
-  sha256: 7e9d7351610eaadddb281e3149a74e22e329bc0b5325f631031d4b63a6a770ae
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 57fabc5f0e45f48d2407b35c731192abc896376411fe7e4bb836ffa03d38f90d
 build:
-  number: 2
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - cython                                 # [build_platform != target_platform]
-    - numpy                                  # [build_platform != target_platform]
     - {{ compiler('c') }}
   host:
+    - cython 0.29.36
+    - numpy {{ numpy }}
     - python
     - pip
     - setuptools
-    - scikit-learn >=0.16
-    - cython >=0.17
-    - joblib
-    - numpy
-
+    - wheel
   run:
     - python
-    - scikit-learn >=0.16
-    - joblib
-    - six
-    - cython >=0.17
-    - {{ pin_compatible('numpy') }}
+    - scikit-learn >=0.20
+    - joblib >=1.0
+    - numpy >=1.20
```

Prefer this instead:
```suggestion
    - {{ pin_compatible('numpy') }}
```
This as the advantage of setting the minimum version to the version that was resolved at build time, and does set  an upper bound.

---

done